src/components/List/List.jsx

```diff
@@ -43,18 +46,27 @@ const List = () => {
       )}
 
       {value && !value.empty && (
-        <ul>
-          {value.docs.map((doc) => (
-            <li key={doc.id}>
-              <input
-                type="checkbox"
-                checked={has24HoursPassed(doc.data().lastDate)}
-                onChange={() => handleCheck(doc.id, doc.data().lastDate)}
-              />
-              {doc.data().name}
-            </li>
-          ))}
-        </ul>
+        <>
+          <input value={filter} onChange={(e) => setFilter(e.target.value)} />
```

Comment:
What do you think if put this input inside of the tag form, so, when we finish typing, we could hit enter :D

Reply:
Hey @FauricioValencia I tried to do that but it would not prevent the submission of the form with enter, only with a button with type submit, I looked up some solutions and the one that made sense, which involved conditionally calling the prevent function with e.key === 'Enter', did not work. Could you shine some light on this please?